apps/jobs/utils.py

```diff
@@ -133,3 +135,36 @@ def get_remaining_submission_for_a_phase(
             "remaining_submissions_count": remaining_submission_count,
         }
         return response_data, status.HTTP_200_OK
+
+
+def is_url_valid(url):
+    """
+    Checks that a given URL is reachable.
+    :param url: A URL
+    :return type: bool
+    """
+    request = urllib.request.Request(url)
+    request.get_method = lambda: 'HEAD'
+    try:
+        urllib.request.urlopen(request)
+        return True
+    except urllib.request.HTTPError:
+        return False
+
+
+def get_file_from_url(url):
+    """ Get file object from a url """
+
+    BASE_TEMP_DIR = tempfile.mkdtemp()
+    file_name = url.split("/")[-1]
+    file_path = os.path.join(BASE_TEMP_DIR, file_name)
+    file_obj = {}
+    headers = {'user-agent': 'Wget/1.16 (linux-gnu)'}
+    r = requests.get(url, stream=True, headers=headers)
```

Comment:
Please change variable `r` to `response`.

Reply:
Done